pkg/kubeapiserver/options/authentication.go

```diff
@@ -170,6 +173,18 @@ func (s *BuiltInAuthenticationOptions) Validate() []error {
 			allErrors = append(allErrors, fmt.Errorf("service-account-issuer contained a ':' but was not a valid URL: %v", err))
 		}
 	}
+	if s.ServiceAccounts != nil && utilfeature.DefaultFeatureGate.Enabled(features.SecureServiceAccounTokenVolume) {
+		if !utilfeature.DefaultFeatureGate.Enabled(features.TokenRequest) || !utilfeature.DefaultFeatureGate.Enabled(features.TokenRequestProjection) {
+			allErrors = append(allErrors, errors.New("If the SecureServiceAccounTokenVolume feature is enabled,"+
+				" the TokenRequest and TokenRequestProjection features must also be enabled"))
+		}
+		if len(s.ServiceAccounts.Issuer) == 0 {
+			allErrors = append(allErrors, errors.New("service-account-issuer is a required flag"))
+		}
+		if len(s.ServiceAccounts.KeyFiles) == 0 {
```

Comment:
and `--service-account-signing-key-file`, right?

Reply:
Added to the error message, I need to figure out how to coallesce this validation with the validation we do in cmd/kube-apiserver, but this has the intended affect.